### ISSUE TITLE
Create conditional logic for home blocks that shows CPT blocks only when CPT exists

### ISSUE THREAD

**Issue by postphotos** [xwp/wp-travel-theme#28]
Since these blocks depend on the CPT and its taxonomies, @miina raised a good point: If there were no taxonomy terms, we should hide these blocks (activities etc.).

Depends on #27.

**Comment by miina** [comment]
@postphotos Feels like it might make sense to implement this within the relevant blocks, thoughts?

**Comment by miina** [comment]
@postphotos Since we're not going to have CPT blocks for now, perhaps this ticket can be closed? Thoughts?

cc @kienstra

**Comment by DavidCramer** [comment]
@postphotos - #38 implements the structures of the theme, and has the features ( taxonomies, CPT, and Blocks ) So, the blocks won't exist if the theme features are not enabled, in the event that it is separated to a standalone plugin. Therefore this is technically already done.
cc @kienstra @miina

**Comment by postphotos** [comment]
Thanks, @DavidCramer and @miina.
